src/tools/business/template_renderer.py

```
@@ -0,0 +1,70 @@
+    *,
+    template_override: Any,
+    default_template: str,
+    variables: Dict[str, Any],
+    call_id: str,
+    tool_name: str,
+) -> str:
+    override = _normalize_template(template_override)
+    if override:
+        try:
+            return render_html_template(html_template=override, variables=variables)
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)